db/models.py

```diff
@@ -1 +1,12 @@
 from django.db import models
+
+
+class Genre(models.Model):
+
```

Comment:
these empty lines seem kinda redundant

Reply:
I have deleted inner blank lines but I can't decrease amount of blank lines between two classes block, flake8 is screaming.